include/exiv2/futils.hpp

```diff
@@ -177,6 +177,9 @@ namespace Exiv2
     //! @brief Return the path of the current process.
     EXIV2API std::string getProcessPath();
 
+    //! @brief Return vector of libraries in memory.
+    EXIV2API std::vector<std::string> getLoadedLibraries();
```

Comment:
There's a `#include <vector>` missing at the top of this file.

Reply:
It's in futils.cpp.

There's something really screwed up with this PR.  I'll delete it and try again later.  I don't want to delete it at this moment because I want to sure that I don't loose the work and I don't have time to work on it at the moment.  I'll deal with this tonight.